### PR TITLE
Epub reader design polishing

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
image

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by JanAckermann** [comment]
Screenshot outdated

File: packages/web-app-epub-reader/src/App.vue [hunk 1]
```diff
@@ -1,14 +1,18 @@
 <template>
-  <div class="epub-reader oc-flex oc-p-s">
-    <oc-list class="epub-reader-chapters-list oc-visible@l">
+  <div class="epub-reader oc-flex">
+    <oc-list class="epub-reader-chapters-list oc-pl-s">
```
Comment (kulmann) [inline packages/web-app-epub-reader/src/App.vue:3]: Why did you decide to remove the `oc-visible@l`? With that the chapter nav sidebar is quite distorted on smaller screens, see screenshots.
Screenshot 2024-02-13 at 22 12 08
Screenshot 2024-02-13 at 22 12 19
Screenshot 2024-02-13 at 22 12 32

Comment (JanAckermann) [reply packages/web-app-epub-reader/src/App.vue:3]: Hm yes. I think that's something I need to sort out when solving the iPad optimization task 🤔 
But we can bring that back in the meantime.

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**